### PR TITLE
Feat/97/preference indicator

### PR DESCRIPTION
**Pull request by raultel**
There's a little ">" indicating which preference you are tweaking.
I also used a dictionary for better code legibility (first commit), so configs[0] is now configs['velocidade']
Lastly, sometimes the indicator does not match the setting if you press the key too fastly, but this probably has to do with the key event logic.

### CODE REVIEW

File: coral.py [hunk 1]
```diff
@@ -291,32 +305,42 @@ def config_prompt():
...
+                        configs[opcao] = 0
                     else:
-                        configs[n] += 1
+                        configs[opcao] += 1
                     if n == 3:
                         update_volume()
                 elif event.key == pygame.K_LEFT:  
-                    if configs[n] == 0:
-                        configs[n] = 2
+                    if configs[opcao] == 0:
```
Comment (raifernando) [inline coral.py:325]: The game crashes (error in line 325) when you open the game and try to change a setting using the left arrow. My guess is that 'opcao' is not defined in that scope.

**Review by raifernando — CHANGES_REQUESTED** [review]
Please fix the crash

File: coral.py (hunk 1 above)
Comment (raultel) [reply coral.py:326]: Should be working now! thanks for pointing this out
Don't know how it didn't cross my mind testing the left arrow key lol

Comment (raifernando) [reply coral.py:326]: Nice! It's working now

**Review by raifernando — APPROVED** [review]
lgtm!

**Comment by rodrigo-lex** [comment]
It seems there are conflicts that need to be resolved before a merge can be done.

**Comment by raultel** [comment]
I think it's working now!

**Pull request merged**